pkg/scheduler/testing/framework/fake_listers.go

```diff
@@ -94,7 +94,7 @@ func (f ControllerLister) GetPodControllers(pod *v1.Pod) (controllers []*v1.Repl
 		}
 	}
 	if len(controllers) == 0 {
-		err = fmt.Errorf("could not find Replication Controller for pod %s in namespace %s with labels: %v", pod.Name, pod.Namespace, pod.Labels)
+		err = errors.NewNotFound(v1.Resource("replicationcontrollers"), pod.Name)
```

Comment:
Apparently, GetXXXs (plural) don't return not-found errors.
e.g., https://github.com/kubernetes/client-go/blob/84a6fe7e4032ae1b8bc03b5208e771c5f7103549/listers/core/v1/replicationcontroller_expansion.go#L62

Reply:
This was just the place to get lost. Thanks for telling me about it. I've learnt 🙏

Reply:
Fixed 74e076d724aa195677522c623b156a3dff04a8bf